http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1390,7 +1392,25 @@ object LeoAppServiceInterp {
     val bSplited = b.asString.split("\\.")
     aSplited(0) == bSplited(0) && aSplited(1) == bSplited(1)
   }
+
+  private[http] def checkIfCanBeDeleted(appType: AppType, appStatus: AppStatus): Either[String, Unit] = {
+    val deletable =
+      appType match {
+        case AppType.Galaxy =>
+          AppStatus.deletableStatuses.contains(appStatus)
+        case _ =>
+          // As of 10/26/2023.
```

Comment:
This is where I would put the great comment that you have above, I feel like it is some very important context that will help future devs to hopefully relax the constraint 🤞